contrib/libhdfs3-cmake/CMakeLists.txt

```diff
@@ -157,6 +157,9 @@ set_source_files_properties("${HDFS3_SOURCE_DIR}/rpc/RpcClient.cpp" PROPERTIES C
 add_library(_hdfs3 ${SRCS})
 
 add_dependencies(_hdfs3 protoc)
+target_compile_options(_hdfs3 PRIVATE
+    -include "${ClickHouse_SOURCE_DIR}/contrib/google-protobuf/src/google/protobuf/stubs/port.h"
```

Comment:
So this header-includes a file, right? Not very pretty to do this via CMake. What is the reason for that? Is it because 3rd-party deps can't include headers from other 3rd party deps? Maybe you can try with `target_include_directories` instead?

Reply:
The issue is that orc and hdfs used to include some headers from protobuf transitively (so they imported just the protobuf-related headers), but are also relying on `google::protobuf::uint8` (or similar) types. After the update the header containing those types does not come transitively anymore, so I had to force-include it with compiler flags.